src/telemetry.ts

```diff
@@ -0,0 +1,73 @@
+import { Env } from "./common/environment";
+import { reactAI } from "react-appinsights";
+import history from "./history";
+import { ApplicationInsights, SeverityLevel } from "@microsoft/applicationinsights-web";
+import { version } from "../package.json";
+import { isElectron } from "./common/hostProcess";
+import { Action } from "redux";
+import { ErrorCode } from "./models/applicationState";
+import { Error } from "tslint/lib/error";
+
+// vott-app-insights
+let INSTRUMENTATION_KEY = "0b9e5117-c78d-40c9-9338-921092cde49a";
```

Comment:
thinking we should dynamically fill this in via Web pack.  It's just an identifier though.

Reply:
moving the key to `.env*`